elements/pl-matrix-output/pl-matrix-output.mustache

```diff
@@ -8,6 +8,7 @@
     <div class="card-header">
         <ul class="nav nav-tabs card-header-tabs" role="tablist">
             <li class="nav-item" role="presentation"><a class="nav-link{{#default_is_matlab}} active{{/default_is_matlab}}" href="#matlab-{{uuid}}" aria-controls="matlab-{{uuid}}" role="tab" data-toggle="pill">matlab</a></li>
+            <li class="nav-item" role="presentation"><a class="nav-link{{#default_is_mathematica}} active{{/default_is_mathematica}}" href="#mathematica-{{uuid}}" aria-controls="mathematica-{{uuid}}" role="tab" data-toggle="pill">mathematica</a></li>
```

Comment:
Can we capitalize the user-visible tab labels to "Mathematica", "Matlab", "Python"?

Reply:
Capitalized